selinux/selinux-autorelabel-generator

```diff
@@ -40,7 +40,9 @@ enable_units() {
 
 		[Service]
 		Type=oneshot
+   		ExecStartPre=/bin/sh -c "echo SELinux relabeling ${realdir}"
 		ExecStart=/sbin/restorecon -R ${opts} ${realdir}
+                StandardOutput=tty
```

Comment:
That means it no longer ends up in the journal. At least `journal+console`

Reply:
Fair enough, change incoming!